cgi-bin/LJ/DB.pm

```diff
@@ -263,7 +263,7 @@ sub foreach_cluster {
     die $@ if $@;
 
     foreach my $cluster_id (@LJ::CLUSTERS) {
-        my $dbr = ($LJ::IS_DEV_SERVER) ?
+        my $dbr = %LJ::CLUSTER_PAIR_ACTIVE ?
```

Comment:
Does this do what we want? This will, on prod, make it call `LJ::DBUtil->get_inactive_db` which immediately calls out to do the a/b role selection and inversion, which won't work in our environment.

I think that this can just return `LJ::get_cluster_reader` in all cases, at least for Dreamwidth. For people who are still using an actual master-master setup (which we haven't in years) then it won't do the right thing, but honestly, I think we can drop support for master-master.